src/app/games/awale/AwaleTutorial.ts

```diff
@@ -19,8 +19,8 @@ export class AwaleTutorial extends Tutorial {
         ),
         TutorialStep.anyMove(
             $localize`Big sowing`,
-            $localize`You are still the first player.
-        When there are enough seeds to make a full turn, something else happens.<br/><br/>
+            $localize`When there are enough seeds to make a full turn, something else happens.<br/><br/>
+        You are playing Dark.
```

Comment:
On choisi you are ou you're :D ?

Reply:
88 you're vs. 3 you are, passons à 91 you're !